agent_build_refactored/tools/run_in_ec2/boto3_tools.py

```diff
@@ -0,0 +1,443 @@
+# Copyright 2014-2022 Scalyr Inc.
```

Comment:
This mainly just a refactored `tests/end_to_end_tests/run_in_remote_machine/ec2.py`, Since we now use ec2 instances during the build process too, it was moved from tests to build packages.